backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -159,6 +160,15 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
             row.getOrderingProviderZipCode().getValue(),
             null);
 
+    List<PhoneNumber> patientPhoneNumbers =
+        row.getPatientPhoneNumber().getValue() != null
+            ? List.of(new PhoneNumber(PhoneType.MOBILE, row.getPatientPhoneNumber().getValue()))
+            : emptyList();
+    List<String> patientEmails =
+        row.getPatientEmail().getValue() != null
+            ? List.of(row.getPatientEmail().getValue())
+            : emptyList();
```

Comment:
check should account for empty string
![Screen Shot 2023-05-19 at 3 38 09 PM](https://github.com/CDCgov/prime-simplereport/assets/7085658/9585a710-cdee-40e0-a376-c035290529c6)

Reply:
might make more sense to fix this behavior in the getValue method
https://github.com/CDCgov/prime-simplereport/blob/617b05e8afb83b272eb8caef329214d2033a951e/backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java#L268-L277

Reply:
good catch, updated that too